sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/encoders/ExpressionEncoderSuite.scala

```diff
@@ -370,6 +370,15 @@ class ExpressionEncoderSuite extends CodegenInterpretedPlanTest with AnalysisTes
     assert(e.getMessage.contains("Cannot use null as map key"))
   }
 
+  test("tuple with more than 22 elements") {
```

Comment:
nit. `throw exception for tuples with more than 22 elements`?

Reply:
Actually either would be fine for me, but I also think that would be more clearer, so just addressed it.